src/java.base/share/classes/sun/security/tools/keytool/Resources_zh_CN.java

```diff
@@ -71,6 +71,8 @@ public class Resources_zh_CN extends java.util.ListResourceBundle {
                 "\u5DF2\u751F\u6210{0}\u5BC6\u94A5"}, //-genseckey
         {"Generated.keysize.bit.keyAlgName.secret.key",
                 "\u5DF2\u751F\u6210 {0} \u4F4D{1}\u5BC6\u94A5"}, //-genseckey
+        {"key.algorithm.weak", "%1$s \u4F7F\u7528\u7684 %2$s \u7B97\u6CD5\u88AB\u89C6\u4E3A\u5B58\u5728\u5B89\u5168\u98CE\u9669\u3002"},
+        {"key.size.weak", "%1$s \u4F7F\u7528\u7684 %2$s \u5B58\u5728\u5B89\u5168\u98CE\u9669\u3002"},
```

Comment:
I can manually replace these translations with one or the other. However, it's determine which parts of the code translate to 被视为存在安全风险" or "存在安全风险". I converted the unicode you highlighted above to characters, and I see where each one is located. If I just replace all instances of "存在安全风险" and insert "被视为存在安全风险" in unicode in its place, will it be OK?

Reply:
Hi @wangweij , I believe I fixed the occurrences of "存在安全风险" and replaced them with "被视为存在安全风险". This translation drop does not update `src/jdk.jartool/share/classes/sun/security/tools/jarsigner/Resources.java`. So, this may be a separate request. If that file is selected for extraction in the next drop in January, then it could also be fixed then. 

Could you re-review the current changes for this drop?